runtime/colors/blue.vim

```diff
@@ -18,6 +18,13 @@ let s:t_Co = &t_Co
 if (has('termguicolors') && &termguicolors) || has('gui_running')
   let g:terminal_ansi_colors = ['#000000', '#cd0000', '#00cd00', '#cdcd00', '#0000ee', '#cd00cd', '#00cdcd', '#e5e5e5', '#7f7f7f', '#ff0000', '#00ff00', '#ffff00', '#5c5cff', '#ff00ff', '#00ffff', '#ffffff']
 endif
+
+if exists('g:terminal_ansi_colors')
+  for color in g:terminal_ansi_colors
+    " convert to Nvim's highlight groups for terminal buffers
+    execute 'let g:terminal_color_' .. index(g:terminal_ansi_colors, color) .. ' = ' .. 'color'
```

Comment:
hmm, using `index()` like this would break if the same colour is used twice (and would need to linearly search the array).

something like:
```vim
for i in range(g:terminal_ansi_colors->len())
  let g:terminal_color_{i} = g:terminal_ansi_colors[i]
endfor
```
may be better?

Reply:
> may be better?

Iterating over `items()` is much more compact but, unfortunately, Neovim doesn't (yet) support calling `items(list)` .

Reply:
Great suggestion @seandewar I didn't even know that I could do `{i}`. I've updated them all to do it your way.